doc/user/content/cloud/quickstart.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Materialize Cloud Quickstart"
+description: "Set up a Materialize Cloud account, create deployments, and connect to data sources."
+menu:
+  main:
+    parent: "cloud"
+    weight: 2
+---
+
+{{< cloud-notice >}}
+
+
+To get you started, we'll walk you through the following:
+
+1. Signing up for Materialize Cloud.
+2. Creating a Materialize Cloud deployment.
+3. Connecting to the deployments from a terminal.
+4. Connecting to a data source.
+5. Running sample queries.
+
+## Sign up for Materialize Cloud
+
+Sign up at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
+
+## Create and connect to a Materialize Cloud deployment
+
+Once you [sign up](https://cloud.materialize.com/signup) for Materialize Cloud and [log in](https://cloud.materialize.com), you use the Cloud management screen to create, upgrade, or destroy deployments, and to obtain the TLS CA you need to install on your local machine.
+
+By default, you can create up to two deployments. If you're interested in more, [let us know](../support).
+
+1. Click **Create deployment**. A deployment is created with an assigned name and hostname.
```

Comment:
Before this gets merged, can you do a last pass over the docs for passive voice? I think really any alternative is better: "We'll create a deployment...", "You'll see a deployment...", "You'll know your deployment is ready when..." etc. etc.

Reply:
Done.